backend/test_observer/common/permissions.py

```diff
@@ -50,6 +50,7 @@ class Permission(StrEnum):
     change_test = auto()
     view_rerun = auto()
     change_rerun = auto()
+    change_rerun_bulk = auto()
```

Comment:
Note: We'll need to update certification-ops to ignore this permission to not break existing workflows